modules/models/email.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+require_once(dirname(__FILE__)."/record.php");
+
+class Email extends Record
+{
+    protected static $table_name = "emails";
+
+    public $id;
+    public $subject;
+    public $content;
+    public $created_at;
+    public $updated_at;
+
+    // Returns the email address of the person who sent this email
+    public function get_sender()
+    {
+        $raw_sql = "
+        SELECT email_address
+        FROM sent
+        WHERE message_id = {$this->id};
+        ";
```

Comment:
was there an absolute need to do this? 🤔 couldn't you have done something like an email `belongs_to` a sender or something like that